pom.xml

```diff
@@ -20,7 +20,7 @@
   <groupId>co.cask.http</groupId>
   <artifactId>netty-http</artifactId>
   <version>0.12.0-SNAPSHOT</version>
-  <packaging>jar</packaging>
+  <packaging>bundle</packaging>
```

Comment:
To double confirm, I added the netty-http dependency as follows after building it as a bundle:

&lt;dependency&gt;
            &lt;groupId>co.cask.http&lt;/groupId&gt;
            &lt;artifactId>netty-http&lt;/artifactId&gt;
            &lt;version>${netty-http.version}&lt;/version&gt;
            &lt;type>jar&lt;/type>
        &lt;/dependency>

Everything works fine, so this change doesn't break anything.